app/src/main/java/com/example/mykotlincalculator21/utils/OperandUtils.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.mykotlincalculator21.utils
+
+import com.example.mykotlincalculator21.utils.NumbersUtils.ZERO_NUMBER_DOUBLE_TYPE
+import com.example.mykotlincalculator21.utils.StringUtils.EMPTY_STRING
+
+class OperandUtils {
+    var signs = EMPTY_STRING
+    var value = EMPTY_STRING
+
+    fun eraseOperands() {
+        signs = EMPTY_STRING
+        value = EMPTY_STRING
+    }
+
+    fun getValue(): Double {
+        if (value === EMPTY_STRING) {
```

Comment:
=== operator is used to compare the reference of two variable or object. It will only be true if both the objects or variables pointing to the same object. The negated counterpart of === in Kotlin is !== which is used to compare if both the values are not equal to each other. For values which are represented as primitive types at runtime (for example, Int), the === equality check is equivalent to the == check.

Reply:
Thx for the tip, now its done